tests/generate_nmd_traj_test.py

```diff
@@ -27,8 +28,11 @@ def setUp(self):
         self.root_dir = tempfile.TemporaryDirectory()
         self.root_path = Path(self.root_dir.name)
 
+        warnings.simplefilter("ignore")
```

Comment:
This allows us to remove the noise coming from incomplete trajectory information being written to a PDB. There's nothing for us to do with that info.